common/element_handle.go

```diff
@@ -23,8 +23,10 @@ import (
 	"github.com/grafana/xk6-browser/k6ext"
 )
 
-const resultDone = "done"
-const resultNeedsInput = "needsinput"
+const (
+	resultDone       = "done"
+	resultNeedsInput = "needsinput"
+)
```

Comment:
Not sure why i'm having to do this in this PR, which is unrelated 🤷

Reply:
I caused this issue. I apologize for that.

Reply:
No worries 😄